server/etcdserver/api/v3alarm/alarms.go

```diff
@@ -88,16 +82,10 @@ func (a *AlarmStore) Deactivate(id types.ID, at pb.AlarmType) *pb.AlarmMember {
 
 	delete(t, id)
 
-	v, err := m.Marshal()
-	if err != nil {
-		a.lg.Panic("failed to marshal alarm member", zap.Error(err))
-	}
-
-	b := a.bg.Backend()
-	b.BatchTx().Lock()
-	b.BatchTx().UnsafeDelete(buckets.Alarm, v)
-	b.BatchTx().Unlock()
-
+	tx := a.bg.Backend().BatchTx()
+	tx.Lock()
+	defer tx.Unlock()
+	buckets.MustUnsafeDeleteAlarm(a.lg, tx, m)
```

Comment:
Please put it together as `buckets.MustDeleteAlarm(...)`

Reply:
Done